dist/info/3dengine_libretro.info

```diff
@@ -9,3 +9,4 @@ license = "GPLv3"
 permissions = ""
 display_version = "v1"
 supports_no_game = "false"
+notes = "(!) 3DEngine does not use any BIOS files."
```

Comment:
When you're on a road, you see a car in front of you with no plate. You know it's missing a plate. That's just real world with standards everywhere.

In digital world, it's not possible to tell if you're missing a plate (or bios) because that note is not there. Things changes too fast in digital world.

I'm more than happy to write out notes. To help standardize all core information.